sql/core/src/test/resources/sql-tests/results/timestampNTZ/datetime.sql.out

```diff
@@ -146,7 +146,7 @@ NULL	2016-12-31	2016-12-31
 -- !query
 select to_timestamp(null), to_timestamp('2016-12-31 00:12:00'), to_timestamp('2016-12-31', 'yyyy-MM-dd')
 -- !query schema
-struct<to_timestamp(NULL):timestamp,to_timestamp(2016-12-31 00:12:00):timestamp,to_timestamp(2016-12-31, yyyy-MM-dd):timestamp>
+struct<to_timestamp(NULL):timestamp_ntz,to_timestamp(2016-12-31 00:12:00):timestamp_ntz,to_timestamp(2016-12-31, yyyy-MM-dd):timestamp_ntz>
```

Comment:
From PR's description:
> Does this PR introduce any user-facing change?
> No, the new timestmap type is not release yet and the default value of "spark.sql.timestampType" is TIMESTAMP_LTZ

But the changes in the golden file seems do change the behavior, doesn't it?

Reply:
updated